src/index.js

```diff
@@ -1,31 +1,87 @@
+          <input type='checkbox' onclick='completedTask(${todo.completed})' >
+          <input value= "${todo.description}" type= "text" class="user-task"/>
+     </div>
+     <div class= "edit-remove" >
+       <ul>
+       <li onclick="removeTask(${todo.index})"><span class="material-symbols-outlined">
+       delete
+       </span></li>
+       </ul>
+       </div>
+       </div>
+      <div class="line"></div>
+       `;
 
-    tasks.forEach((task) => Display.addTaskToList(task));
-  }
+  todoLists.appendChild(displayTasks);
+};
 
-  static addTaskToList(task) {
-    const todoLists = document.querySelector('.tasks-list');
-    const displayTasks = document.createElement('div');
-    displayTasks.innerHTML = `
-   <div class = "todo-list" id = ${task.index} complete = ${task.completed}>
-   <div class = "checkbox-description-container">
-        <input type='checkbox'>
-        <p>${task.description}</p>
-   </div>
-     <i class="fa-solid fa-ellipsis-vertical"></i>
-     </div>
-    <div class="line"></div>
-     `;
-    todoLists.appendChild(displayTasks);
+inputField.addEventListener('keypress', (e) => {
+  const todoLists = inputField.value;
+  if (e.key === 'Enter' && todoLists) {
+    if (localStorage.getItem('duties') === null) {
+      taskLists = [];
+    } else {
+      taskLists = JSON.parse(localStorage.getItem('duties'));
+    }
+    const assignment = {
+      description: todoLists,
+      completed: false,
+      index: taskLists.length + 1,
+    };
+    e.target.value = '';
+    taskLists.push(assignment);
+    localStorage.setItem('duties', JSON.stringify(taskLists));
+    tasksTodo(assignment);
   }
+});
+
+window.removeTask = (index) => {
+  const taskId = document.getElementById(index);
+  const deleteTask = JSON.parse(localStorage.getItem('duties'));
+  const newTodo = deleteTask.filter((task) => task.index !== index);
+  newTodo.forEach((task, i) => {
+    task.index = i + 1;
+  });
+  localStorage.setItem('duties', JSON.stringify(newTodo));
+  taskId.remove();
+};
+
+const tasks = JSON.parse(localStorage.getItem('duties')) || [];
+if (localStorage.getItem('duties')) {
+  tasks.forEach((task) => {
+    tasksTodo(task);
+  });
 }
 
-document.addEventListener('DOMContentLoaded', Display.showTasks);
+const editTasks = document.querySelectorAll('.user-task');
+editTasks.forEach((task, index) => {
+  task.addEventListener('change', (e) => {
+    const updatedTask = e.target.value;
+    taskLists = JSON.parse(localStorage.getItem('duties'));
+    taskLists[index].description = updatedTask;
+    localStorage.setItem('duties', JSON.stringify(taskLists));
+  });
+});
+editTasks.forEach((task, index) => {
+  task.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      const updatedTask = e.target.value;
+      taskLists = JSON.parse(localStorage.getItem('duties'));
+      taskLists[index].description = updatedTask;
+      localStorage.setItem('duties', JSON.stringify(taskLists));
+    }
+  });
```

Comment:
- [ ] Adding and removing functionalities are working just fine, but I couldn't edit the task after adding them. Kindly add the function to edit task description, it is a project requirement for a better user experience. 👆🏽

- [ ] **_[OPTIONAL]_** It is ok that you try to implement the function for the Clear All button, however, it is not working because the status of "complete" is false even though the check button is checked. Please 🙏🏽 correct this function or remove it until the project requirements demand it.